flink-test-utils/src/main/java/org/apache/flink/test/util/TestEnvironment.java

```diff
@@ -80,7 +80,7 @@ public String getExecutionPlan() throws Exception {
 	private OptimizedPlan compileProgram(String jobName) {
 		Plan p = createProgramPlan(jobName);
 
-		Optimizer pc = new Optimizer(new DataStatistics());
+		Optimizer pc = new Optimizer(new DataStatistics(), this.executor.getConfiguration());
```

Comment:
The executor here is of type `ForkableFlinkMiniCluster`. We have too many local test clusters in Flink :(